command/results_go_test2json.go

```diff
@@ -45,13 +45,18 @@ func (c *goTest2JSONCommand) Execute(ctx context.Context,
 		return errors.Wrap(err, "failed to expand files")
 	}
 
-	catcher := grip.NewBasicCatcher()
-	for i := range c.Files {
-		if ctx.Err() != nil {
-			return errors.New("operation canceled")
-		}
+	// All file patterns should be relative to the task's working directory.
+	for i, file := range c.Files {
+		c.Files[i] = filepath.Join(conf.WorkDir, file)
+	}
 
-		catcher.Add(c.executeOneFile(ctx, c.Files[i], comm, logger, conf))
+	files, err := globFiles(c.Files...)
+	if err != nil {
+		return errors.Wrapf(err, "obtaining names of output files")
```

Comment:
This is a strange error message, can you add "problem" or "error" to the start?

Reply:
I actually pretty much never add "problem"/"error" because usually they're useless noise when all you really need is the context of what errored. Also, when wrapping a bunch of them, they just end up looking like "problem with abc: error doing def: problem doing xyz". I follow the go standard library's convention of just wrapping errors for context.
Style guide: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
Hmm okay I stand corrected.